pkgs/applications/editors/vim/default.nix

```diff
@@ -50,6 +50,15 @@ stdenv.mkDerivation {
     ln -s $out/bin/vim $out/bin/vi
     mkdir -p $out/share/vim
     cp "${vimrc}" $out/share/vim/vimrc
+
+    # Prevent bugs in the upstream makefile from silently failing and missing outputs.
```

Comment:
Are those bugs in an upstream issue tracker somewhere? Could we link to that?

Reply:
If so I'd also link from common.nix in line 13

Reply:
@raboof Here it's hardening from bugs as in future regressions. The two I know about have an upstream fix sent:

 - https://github.com/vim/vim/pull/12288

@dasJ what do you mean? It's not intrinsically linked to parallel building. For example, it's unknown what made https://github.com/NixOS/nixpkgs/issues/227598 fail in a similar way.